x-pack/plugins/triggers_actions_ui/public/application/lib/check_action_type_enabled.tsx

```diff
@@ -45,7 +44,9 @@ export function checkActionTypeEnabled(
             {
               defaultMessage: 'This feature requires a {minimumLicenseRequired} license.',
               values: {
-                minimumLicenseRequired: actionType.minimumLicenseRequired,
+                minimumLicenseRequired: `${actionType.minimumLicenseRequired
```

Comment:
Do the license levels not need to be translated?  If not, I'd bet there's an "initial cap" function somewhere in a npm module that Kibana includes, I guess it might be hard to find though.

Reply:
Awesome!